### PR TITLE
Adding ability to `ls -s backlog`

### PR DESCRIPTION
**Pull request by kedark3**
`ls -s backlog` which will find all issues in the backlog.

### CODE REVIEW

File: simplejira/wrapper.py [hunk 1]
```diff
@@ -315,11 +315,14 @@ def search_issues(self, assignee=None, sprint=None, status=None, text=None):
         Returns:
             List of JIRA.Issue resources
         """
-        if not sprint:
-            sprint = self.current_sprint_id
+        if sprint == "backlog":
+            search_query = 'project = RHCFQE AND sprint is EMPTY'
```
Comment (bsquizz) [inline simplejira/wrapper.py]: We don't want to hard code project. You can use 'self.project_id' in the wrapper to get current project ID. I think the search will work with the id number as well?

Comment (bsquizz) [reply simplejira/wrapper.py]: (project_id is a property ... look for 'def project_id')

Comment (kedark3) [reply simplejira/wrapper.py]: Updated. Please review.

**Comment by bsquizz** [comment]
Looks good

**Pull request merged**